File: tests/scripts/subcommands/compile_and_run/stage-with-grader/grader/py/grader.py
import sys
import add1

def main():
    a, b = [int(x) for x in input().split()]
    sys.stdin.close()

    res = add1.solve(a, b)

    # BEGIN SECRET
    print("OK")
    # END SECRET
    print(res)
    sys.stdout.close()

if __name__ == '__main__':
    main()

File: tests/scripts/subcommands/compile_run_crun/stage-with-grader/grader/cpp/grader.cpp
#include "add1.h"
#include <cassert>
#include <cstdio>

int main() {
	int a, b;
	assert(2 == scanf("%d%d", &a, &b));
	fclose(stdin);

	int res = solve(a, b);

	// BEGIN SECRET
	printf("OK\n");
	// END SECRET
	printf("%d\n", res);
	fclose(stdout);
	return 0;
}
